### PR TITLE
Updated .gitignore to reflect v1.9rc changes

### PR DESCRIPTION
**Pull request by NaincyKumariKnoldus**
Hey! I have updated .gitignore to reflect v1.9rc changes. Please Review this !
Thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,12 @@ workflows/rnaseq/downstream/*tsv
 workflows/chipseq/data
 workflows/rnaseq/data
 workflows/colocalization/results
+workflows/rnaseq/downstream/functional-enrichment.html
+workflows/rnaseq/downstream/functional-enrichment_cache/
+workflows/rnaseq/downstream/functional-enrichment_files/
+workflows/rnaseq/downstream/Rplots.pdf
+*.Rds
+*.Rd (covers new additions to lib/lcdbwf/)
```
Comment (daler) [inline .gitignore:22]: Only the `*.Rd` is needed on this line.

Comment (daler) [reply .gitignore]: (never mind, I just fixed this while resolving the conflict)

**Comment by daler** [comment]
Thanks. I edited the PR to point to v1.9rc, and there was a conflict but I just resolved that.

**Comment by daler** [comment]
Also, this resolves #358

**Comment by daler** [comment]
@NaincyKumariKnoldus the tests are failing because they are not currently set up to properly handle pull requests from forks. There are also some changes that should be made here that more accurately reflect the current state of v1.9rc. I will make those changes here and then merge without fixing the tests, and will separately need to work on supporting tests from forks.

**Comment by NaincyKumariKnoldus** [comment]
Thanks a lot!

**Pull request merged**